.circleci/config.yml

```diff
@@ -26,6 +26,41 @@ jobs:
       - store_test_results:
           path: /tmp/test-results
 
+  cli-functional-test:
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
This is only minor: Can we use `ubuntu-1604:202004-01` from https://circleci.com/docs/2.0/configuration-reference/#available-machine-images to have a better match up to devs machines?